mlflow/types/llm.py

```diff
@@ -119,6 +119,9 @@ class ChatParams(_BaseDataclass):
         presence_penalty: (float): An optional param of positive or negative value,
             positive values penalize new tokens based on whether they appear in the text so far,
             increasing the model's likelihood to talk about new topics.
+        metadata (Dict[str, str]): An optional param to provide arbitrary additional context
+            to the model. Both the keys and the values must be strings (i.e. nested dictionaries
+            are not supported).
```

Comment:
The reason we only support is str is we don't support union type?

Reply:
yes that's correct